src/src/components/Contexts/__tests__/CurrentViewContext.test.ts

```diff
@@ -21,7 +21,7 @@ theories(parsePath, [
     },
   },
   {
-    in: ['/calendar/u/0/r/week/', defaultCustomViewSize],
+    in: ['/calendar/u/0/r/week/', defaultCustomViewSize, 0],
```

Comment:
Do you want to add any test cases with weekStart not being `0`?

Reply:
Good point, added